js/src/forum/components/AffixedSidebar.js

```diff
@@ -1,16 +1,12 @@
-/**
- * Setup the sidebar DOM element to be affixed to the top of the viewport
- * using Bootstrap's affix plugin.
- *
- * @param {DOMElement} element
- * @param {Boolean} isInitialized
- * @param {Object} context
- */
-export default function affixSidebar(element, isInitialized, context) {
-  if (isInitialized) return;
-
-  const onresize = () => {
-    const $sidebar = $(element);
+import Component from '../../common/Component';
+
+export default class AffixedSidebar extends Component {
```

Comment:
I seem to remember that the affix behavior was no longer working in the current Flarum version. So this change also fixes it?

Reply:
Maybe I don't understand what affixed sidebar does then? Seems to work for me on discuss.flarum.org on the user page

Reply:
Nevermind, I misremembered. The code I was thinking of is on the TagsPage and wasn't doing anything since the homepage didn't use affix.

https://github.com/flarum/tags/blob/1c6dfdd0ff6ad2bfb8c68baa2ce160f7e926c74f/js/src/forum/components/TagsPage.js#L26

I see it has been removed in the Mithril2 PR for Tags, so all is good. Neither index nor tags page use the affix behavior, that's good.